src/kademlia/dht_storage.cpp

```diff
@@ -84,16 +88,16 @@ namespace
 	{
 		// the actual value
 		std::unique_ptr<char[]> value;
+		// size of malloced space pointed to by value
+		int size = 0;
```

Comment:
I don't want to nit-pick, but now you got 32 bit padding here, and 32 bit padding after ``num_announcers``.